### PR TITLE
bug fix for ARD scenes that have been moved but not archived.

### PR DESCRIPTION
**Pull request by dunkgray**
Plus adding ways to test this.

### CODE REVIEW

File: tests/ard_reprocessed_l1s/dev_go_reprocess.sh [hunk 1]
```diff
@@ -2,6 +2,14 @@
 # If a number is passed in it is assumed to be the scene limit
 # otherwise the default is 400
 
+#PBS -P v10
```
Comment (gordonyeong) [inline tests/ard_reprocessed_l1s/dev_go_reprocess.sh:5]: Would we ever use these flags?

File: tests/ard_reprocessed_l1s/repo_go_reprocess.sh [hunk 2]
```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+# If a number is passed in it is assumed to be the scene limit
+# otherwise the default is 400
+
+#PBS -P v10
```
Comment (gordonyeong) [inline tests/ard_reprocessed_l1s/repo_go_reprocess.sh:5]: would we ever use these flags in PBS?

File: tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py [hunk 3]
```diff
@@ -44,11 +74,9 @@ def set_up_dirs_and_db():
     )
 
 
-def test_scene_move(set_up_dirs_and_db):
+def test_ard_reprocessed_l1s(set_up_dirs_and_db):
```
Comment (gordonyeong) [inline tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py:77]: should we rename this test to have the word, 'move' inline with the comment below (ie. moving towards self documenting code)?

File: tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py [hunk 4]
```diff
@@ -130,4 +145,45 @@ def test_scene_move(set_up_dirs_and_db):
...
+            "blocked_l1_zip_path": "not used",
+            "blocking_ard_zip_path": old_fname_06_27,
+        }
+    ]
+    l1_zips, uuids2archive = move_blocked(
+        blocked_scenes, current_base_path.resolve(), new_base_path.resolve()
+    )
+
+    # Assert the dir has been moved
+    assert os.path.isfile(fname_06_27) == True
```
Comment (gordonyeong) [inline tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py:169]: do we care if the file was zero in byte size?

`
[gy5636@gadi-login-03 usgsdownloader]$ ls -la /tmp/roar
-rw-r--r-- 1 gy5636 u46 0 Jun  6 12:31 /tmp/roar
[gy5636@gadi-login-03 usgsdownloader]$ python3
Python 3.6.8 (default, Nov  8 2022, 11:32:15) 
[GCC 8.5.0 20210514 (Red Hat 8.5.0-15)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> path="/tmp/roar"
>>> file_check = os.path.isfile(path)
>>> print(file_check)
True
>>> 
`

File: tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py [hunk 5]
```diff
@@ -130,4 +145,45 @@ def test_scene_move(set_up_dirs_and_db):
...
+            "blocked_l1_zip_path": "not used",
+            "blocking_ard_zip_path": fname_06_27,
+        }
+    ]
+    l1_zips, uuids2archive = move_blocked(
+        blocked_scenes, current_base_path.resolve(), new_base_path.resolve()
+    )
+
+    # Assert the dir ... is still there
+    assert os.path.isfile(fname_06_27) == True
```
Comment (gordonyeong) [inline tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py:187]: same as above. Do we care if the file size was zero?

File: tests/ard_reprocessed_l1s/db_index.sh [hunk 6]
```diff
@@ -31,7 +36,9 @@ fi
 if [[ $HOSTNAME == *"LAPTOP-UOJEO8EI"* ]]; then
   echo "duncans laptop"
   echo "conda activate /home/duncan/bin/miniconda3/envs/odc2020"
-  ODCCONF="--config ${SCRIPT_DIR}/duncan_local.conf"
+  echo "sudo service postgresql start"
+  echo "This env does not have hd5, so do not do ARD processing"
+  ODCCONF="--config ${SCRIPT_DIR}/${USER}_local.conf"
```
Comment (gordonyeong) [inline tests/ard_reprocessed_l1s/db_index.sh:41]: i noticed we removed the line which refers to the duncan_local.conf file (and that file is added to this review) thus for odcconf, we use the local user's conf  (ie. dsg547_local.conf or gy5636_local.conf) . 
Why is duncan_local.conf being added to the repository?

File: tests/ard_reprocessed_l1s/dev_go_reprocess.sh (hunk 1 above)
Comment (dunkgray) [reply tests/ard_reprocessed_l1s/dev_go_reprocess.sh:5]: These ones are used, when submitting a job to the queue

File: tests/ard_reprocessed_l1s/repo_go_reprocess.sh (hunk 2 above)
Comment (dunkgray) [reply tests/ard_reprocessed_l1s/repo_go_reprocess.sh:5]: These ones aren't, I'll get rid of them

File: tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py (hunk 3 above)
Comment (dunkgray) [reply tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py:77]: I'll fix the comment

File: tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py (hunk 4 above)
Comment (dunkgray) [reply tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py:169]: > do we care if the file was zero in byte size?
not really

File: tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py (hunk 5 above)
Comment (dunkgray) [reply tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py:187]: not really

File: tests/ard_reprocessed_l1s/db_index.sh (hunk 6 above)
Comment (dunkgray) [reply tests/ard_reprocessed_l1s/db_index.sh:41]: This is so tests work locally, with a user name of duncan

File: tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py (hunk 5 above)
Comment (gordonyeong) [reply tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py:187]: i should have said this earlier but another way of making the tests reflect our logic is to do this:

`
does_the_file_persist =  os.path.isfile(fname_06_27)
assert does_the_file_persist == True
`

**Review by gordonyeong — APPROVED** [review]

**Pull request merged**